docs/changelog.rst

```diff
@@ -2,6 +2,8 @@
 Changelog
 =========
 
+* :bug:`2775` FTX assets will now be correctly read if they are in the known asset's list.
```

Comment:
nitpick: Perhaps change in another PR though.

User don't know and don't care what a known assets list is.

```suggestion
* :bug:`2775` Ambiguous FTX assets will now be properly recognized by rotki.
```